### DISCUSSION TITLE
A huge Love letter

### DISCUSSION THREAD

**Discussion by JSTLWK** [tailwindlabs/tailwindcss#3212] [Kind words]
Hello TailwindTeam,

I'm in love with TailwindCSS and all of the CSS framework plugins that have been developed around TailwindCSS. As I am a starter in the Development work and getting my experience I found out that TailwindCSS has actually made me get better at CSS and animations in CSS. 

Along with all the videos and the amazing content that has been created such as 'recreating x with TailwindCSS' it has helped me the most of seeing how something works in tailwind play when you mess a little with CSS. 

You guys deserve more than these words, but you should be proud of, in my opinion, the best CSS framework there is on the market. ❤

**Comment by simonswiss** [comment]
Thank you @Hoopless ❤️
